libs/datamodel/core/src/dml/datamodel.rs

```diff
@@ -73,7 +73,7 @@ impl Datamodel {
     /// Finds a model by database name.
     pub fn find_model_db_name(&self, db_name: &str) -> Option<&Model> {
         self.models()
-            .find(|model| model.database_name.as_deref() == Some(db_name))
+            .find(|model| model.database_name.as_deref() == Some(db_name) || model.name == db_name)
```

Comment:
is this ok?

Reply:
From call: This is used in re-introspection, and there we only want the remapped names. So we need a separate method.